addons/website_slides/views/website_slides_templates_lesson_fullscreen.xml

```diff
@@ -108,6 +108,17 @@
             <t t-foreach="slides" t-as="slide">
                 <t t-set="slide_completed" t-value="channel_progress[slide.id].get('completed')"/>
                 <t t-set="can_access" t-value="can_access_channel or slide.is_preview"/>
+                <t t-set="slide_icon" t-if="slide.slide_type == 'pdf'" t-value="'fa-file-pdf-o'"/>
+                <t t-set="slide_icon" t-elif="slide.slide_type == 'sheet'" t-value="'fa-file-excel-o'"/>
+                <t t-set="slide_icon" t-elif="slide.slide_type == 'doc'" t-value="'fa-file-word-o'"/>
+                <t t-set="slide_icon" t-elif="slide.slide_type == 'slides'" t-value="'fa-file-powerpoint-o'"/>
+                <t t-set="slide_icon" t-elif="slide.slide_type == 'image'" t-value="'fa-file-picture-o'"/>
+                <t t-set="slide_icon" t-elif="slide.slide_type == 'youtube_video'" t-value="'fa-youtube-play'"/>
+                <t t-set="slide_icon" t-elif="slide.slide_type == 'google_drive_video'" t-value="'fa-play-circle-o'"/>
+                <t t-set="slide_icon" t-elif="slide.slide_type == 'vimeo_video'" t-value="'fa-vimeo'"/>
+                <t t-set="slide_icon" t-elif="slide.slide_type == 'article'" t-value="'fa-file-text-o'"/>
+                <t t-set="slide_icon" t-elif="slide.slide_type == 'quiz'" t-value="'fa-question-circle-o'"/>
+                <t t-set="slide_icon" t-else="" t-value="'fa-file-o'"/>
```

Comment:
If I do :
```xml
        <button class="o_wslides_button_complete btn btn-sm" t-if="slide_completed and slide.can_self_mark_uncompleted or not slide_completed and slide.can_self_mark_completed">
            <i t-if="slide_completed" class="o_wslides_slide_completed fa fa-check-circle fa-fw text-success fa-lg" t-att-data-slide-id="slide.id" title="Mark as not done"/>
           <t t-else="" t-call="website_slides.slide_icon" t-att-data-slide-id="slide.id" title="Mark as done"/>
        </button>
```
it works in the `slide_sidebar_done_button` (the done button `Python` template) so when I launch the fullscreen it displays the correct icons.
However it crashes in the JS equivalent `website.slides.sidebar.done.button` (when I click on the icon to toggle it) by saying `Error: QWeb2: Template 'website_slides.slide_icon' not found`.

I think it has something to do with the fact that I cannot call regular data templates in a qweb template because the data template isn't loaded? :thinking: 
So I could potentially do it in the Python one but for the JS one I think I still need the `slide_icon` to be passed in the dom data so I still need it to be `t-set`.

Moreover I have to keep the `t-att-data-slide-id="slide.id"` attribute (`slide.id` in the py and `slideId` in the JS) and the title on the icon (so what I did above with the t-else attributes doesn't work). I'll have to add a variable (t-set) for the slide-id data and for the title to be passed to the `slide_icon` template which complexes it with conditions :thinking: (I don't know if this part is such a problem though)

What do you think? Did I misunderstood or missed something? :thinking: How do you think it could be done?